Source/WebCore/css/CSSSelector.h

```diff
@@ -394,7 +395,7 @@ inline bool CSSSelector::matchesPseudoElement() const
 
 inline bool CSSSelector::isWebKitCustomPseudoElement() const
 {
-    return pseudoElement() == PseudoElement::WebKitCustom || pseudoElement() == PseudoElement::WebKitCustomLegacyPrefixed;
+    return pseudoElement() == PseudoElement::WebKitCustom || pseudoElement() == PseudoElement::WebKitCustomLegacyPrefixed || pseudoElement() == PseudoElement::WebKitUnknown;
```

Comment:
I wouldn't add `pseudoElement() == PseudoElement::WebKitUnknown` in here, especially with the new name. I'd just handle `PseudoElement::WebKitUnknown` as its own case in the only callsite of the function.